### PR TITLE
Added more parsing to cgroup classes

### PR DESCRIPTION
**Pull request by grariel**
Added:

- Ability to get memory usage
- `cpuacct` class that allows getting cpu usage
- `pids_in_cgroup` propery.

### CODE REVIEW

**Comment by grariel** [comment]
Don't CR yet, decided to add more features to this PR.

**Comment by grariel** [comment]
@DanielShaulov ready for CR.

File: granulate_utils/linux/cgroups/base_cgroup.py [hunk 1]
```diff
@@ -55,6 +56,10 @@ def cgroup_mount_path(self) -> Path:
     def cgroup_path(self) -> Path:
         return Path(self.get_cgroup_hierarchies()[self.subsystem])
 
+    @property
```
Comment (DanielShaulov) [inline granulate_utils/linux/cgroups/base_cgroup.py:59]: This is a heavy operation and is not cached - should be a function IMO

File: granulate_utils/linux/cgroups/base_cgroup.py [hunk 2]
```diff
@@ -55,6 +56,10 @@ def cgroup_mount_path(self) -> Path:
...
+    def pids_in_cgroup(self) -> List[int]:
+        return [int(proc) for proc in self.read_from_control_file(self.cgroup_procs).split()]
```
Comment (DanielShaulov) [inline granulate_utils/linux/cgroups/base_cgroup.py:61]: Return a `Set` instead to remove dups?

image

**Review by DanielShaulov — CHANGES_REQUESTED** [review]
Small comments

File: granulate_utils/linux/cgroups/base_cgroup.py (hunk 2 above)
Comment (grariel) [reply granulate_utils/linux/cgroups/base_cgroup.py]: Great catch.

File: granulate_utils/linux/cgroups/base_cgroup.py (hunk 1 above)
Comment (grariel) [reply granulate_utils/linux/cgroups/base_cgroup.py]: Indeed, from PEP8:
> Avoid using properties for computationally expensive operations; the attribute notation makes the caller believe that access is (relatively) cheap.

Fixed.

**Review by DanielShaulov — APPROVED** [review]

**Pull request merged**